src/test/resources/data/court_202012141528.csv

```diff
@@ -1,268 +1,24 @@
 court_name,court_code
-Pontypridd Union Street,B62PM
-Bath,B52BB
-Bath County Court Henry Street,B52VU
-Bristol,B52CM
-Bristol County Court Lewins Mead,B52VV
-Bristol Civil Justice Centre,B52QA
-Crewe,B07ED
```

Comment:
At some point soon, we'll need to restore these lines. Could they be commented ? Might be easier in the long run.